docs/how-to-help-with-video-challenges.md

```diff
@@ -71,11 +71,11 @@ You can find the markdown files for video challenges at the following locations
 
 Pick a challenge markdown file from the options above.
 
-### Skim through the video associated with the challenge and create a mutiple choice question
+### Skim through the video associated with the challenge and create a multiple choice question
 
-First, find the videoId.
+First, find the video Id.
```

Comment:
in this case it is a variable, so `videoId` should be correct correct.

```suggestion
First, find the videoId.
```

Reply:
```suggestion
First, find the `videoId`.
```